### ISSUE TITLE
unknown formatter: terminal256

### ISSUE THREAD

**Issue by cuihq** [jneen/rouge#734]
## Env:

* MRI 2.4.1(latest)
* rouge-2.1.1

## Code:

```ruby
require 'rouge'
Rouge.highlight('var foo = 1;', 'js', 'terminal256')
```

## Result:

> RuntimeError: unknown formatter 
	from /usr/local/bundle/gems/rouge-2.1.1/lib/rouge.rb:29:in `highlight'

please fix/merge it!:smile:

**Comment by gfx** [comment]
This seems not a library bug but a documentation bug.

Use `Rouge.highlight('puts "Hello!"', 'ruby', Rouge::Formatters::Terminal256.new)` instead.

**Comment by gfx** [comment]
Sorry, `Rouge.highlight` must accept the tag of formatters.

**Comment by gfx** [comment]
Fixed in #735 (+ #741 for docs and tests).
